### ISSUE TITLE
Support all files types

### ISSUE THREAD

**Issue by kopeclu2** [m-inan/react-upload-gallery#26]
Hello i need to support all files types. And i dont want to type all the format (insposible), any solutions ? :) 
Thank you so much <3

**Comment by kopeclu2** [comment]
Maybe like accpet={[]} // For all support format ?

**Comment by m-inan** [comment]
You will have to recreate custom cards for data types as cards do not support all files.

```js
<RUG
  accept={[
    'pdf',
    'doc',
  ]}
  acceptType="application"
>
  {(files) => (
    files.map(file => (
      <div>{file.name}</div>
    ))
  )}
</RUG>
```

**Comment by m-inan** [comment]
@kopeclu2 Of course, this is not impossible, but I developed this library for images. so there is ui support for images only. For example, if a txt file is to be uploaded, the url of the txt file will be added to the image. Also in the source code there is a similar usage of `accept={accept.map((type) => acceptType + '/' + type)}` so I will need to adapt the source code for all files. You can fork the repo instead, we can edit it to allow you or I to upload all the files for the input. However, for all possible file types, you should display their name in the box or add an appropriate card or list view for each file type.
